contracts/Archetype.sol

```diff
@@ -183,6 +183,21 @@ contract Archetype is ERC721A__Initializable, ERC721AUpgradeable, ERC721A__Ownab
     mintTo(auth, quantity, msg.sender, affiliate, signature);
   }
 
+  function batchMintTo(
+    Auth calldata auth,
+    address[] calldata to,
+    uint256[] calldata quantity,
+    address affiliate,
+    bytes calldata signature
+  ) public payable {
+    if(quantity.length != to.length) {
+      revert InvalidConfig();
+    }
+    for (uint256 i=0; i< to.length; i++){
+      mintTo(auth, quantity[i], to[i], affiliate, signature);
```

Comment:
Wouldn't it be more gas-efficient to execute `_mint(to, quantity);` in the loop rather than `mintTo`?

`mintTo` performs validation for each address. However, validation only needs to be performed a single time for the auth/affiliate/signature.